FlashDevelop/MainForm.cs

```diff
@@ -1713,6 +1713,24 @@ public void OnDocumentModify(ITabbedDocument document)
             }
         }
 
+        public void OnFileSave(ITabbedDocument document, string oldFile, string reason)
```

Comment:
Why did you decide to not change the old OnFileSave without reason to just call this one?